domain/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
+}
```

```suggestion
plugins {
    id("kotlin")
}
```
도메인 로직을 코틀린으로 작성한다면 꼭 java library로 설정하진 않아도 될 것 같아요! 
> domain 모듈이 순수 코틀린 라이브러리로 분리되어야 한다.